src/lib/api/react/ReactComponent.hx

```diff
@@ -48,6 +48,8 @@ extern class ReactComponentOf<TProps, TState, TRefs>
 	/**
 		https://facebook.github.io/react/docs/component-api.html#setstate
 	**/
+	@:overload(function(nextState: TState -> TProps -> TState, ?callback: Void -> Void): Void {})
```

Comment:
remove space after ":" to match code conventions and its good to get merged, I will assign Philippe to approve